content/en/docs/contributing/development.md

```diff
@@ -28,6 +28,40 @@ Gitpod automatically installs the repo-specific packages for you.
 You're now ready to [build](#build), [serve](#serve) or make updates to the
 website files.
 
+## Local development container setup
```

Comment:
I understand that using VSCode is simpler, but can one use other IDEs or work from the command line? If so, could we separate those instructions?